133 complete chapter human structure 734 (#181)
asam-ev/OpenMATERIAL: content/geometry/object-human/human-general.adoc

= General

A human is a bipedal being that can be represented by an object in 3D geometry.

The human 3D geometry is generically structured into the node structure and skeleton (also often called armature) defined in the xref:../geometry/object-human/human-index.adoc#_model_structure[Model structure].
The structure starts with a Root node as a parent group for all other nodes in the structure.
Splitting the object into different parts, such as clothing, hair, and accessories, enables animation and exchange of parts.
Not all nodes have to be present in every human 3D model.
Custom object parts may be added to facilitate and better visualize additional use cases, but all custom parts shall adhere to the specified structure principle.

> In the coordinate system for human 3D geometry, the x-axis points forwards, the y-axis points sidewards, and the z-axis points upwards.

Reviewer: @ClemensLinnhoff @ipg-sig is this what you wanted?

> In the coordinate system for human 3D geometry, the x-axis points forwards, the y-axis points sidewards, and the z-axis points upwards.

Reviewer 2: I think "in the default pose" can be removed. The y-axis always points in the direction of the bone, in all poses.
So this basically only applies to the x- and z-axis that they are forwards and sideways in the default pose.

> In the coordinate system for human 3D geometry, the x-axis points forwards, the y-axis points sidewards, and the z-axis points upwards.

Author: This only specifies the general coordinate system of the 3D assets for humans, not the orientation of the bones. Therefore I would leave this sentence as it is and add a new one, which specifies the bone direction.

+ In the coordinate system for human 3D geometry, the x-axis points forwards, the y-axis points sidewards, and the z-axis points upwards.
+ The default pose is the A-pose.
+ The naming convention makes it possible to mirror one side of the skeleton to the other.
+ The bones follow a specific bone orientation. The y-axis of a bone is always following the bone direction, regardless if it is pointing sidewards, upwards or forwards. The x-axis is mostly facing forwards.

> In the coordinate system for human 3D geometry, the x-axis points forwards, the y-axis points sidewards, and the z-axis points upwards.

Reviewer 2: That sounds better, thanks!

> In the coordinate system for human 3D geometry, the x-axis points forwards, the y-axis points sidewards, and the z-axis points upwards.

Reviewer: I've updated the file accordingly :)

> In the coordinate system for human 3D geometry, the x-axis points forwards, the y-axis points sidewards, and the z-axis points upwards.

Author: Thank you! :)

The default pose is the A-pose.

> The naming convention makes it possible to mirror one side of the skeleton to the other.

Reviewer: ```suggestion
The naming convention makes it possible to mirror one side of the skeleton to the other.
The bones follow a specific bone orientation: The y-axis of a bone always follows the bone direction, regardless of whether it points sidewards, upwards, or forwards. The x-axis usually faces forwards accordingly.
```

= General

> A human is a bipedal being that can be represented by an object in 3D geometry according to the human body's physical representation.

Reviewer: @ACMackenthun please review :)

> A human is a bipedal being that can be represented by an object in 3D geometry according to the human body's physical representation.

Reviewer 3: ```suggestion
A human is a bipedal being that can be represented by an object in 3D geometry.
```

> The human 3D geometry is generically structured into the node structure and skeleton (also often called armature) defined in the <<Structure overview>>. The structure starts with a Root node as a parent group for all other nodes in the structure. Splitting different object parts, such as clothing, hair, and accessories enables the animation and exchange of parts. Not all nodes have to be present in every human 3D model. Additional object parts may be added to facilitate and better visualize additional use cases. They should adhere to the same structure principle.

Reviewer 3: ```suggestion
The human 3D geometry is generically structured into the node structure and skeleton (also often called armature) defined in the <<Structure overview>>. The structure starts with a Root node as a parent group for all other nodes in the structure. Splitting the object into different parts, such as clothing, hair, and accessories, enables the animation and exchange of parts. Not all nodes have to be present in every human 3D model. Custom object parts may be added to facilitate and better visualize additional use cases, but all custom parts shall adhere to the specified structure principle.
```

> The human 3D geometry is orientated at x as the axis facing forwards, y as the axis facing sidewards, and z as the axis facing upwards. The default pose is the A-pose. The naming convention makes it possible to mirror one side of the skeleton to the other. The bone orientation allows (future) animations to be mirrored an both sides.

Reviewer 3: ```suggestion
In the coordinate system for human 3D geometry, the x-axis points forwards, the y-axis points sidewards, and the z-axis points upwards. The default pose is the A-pose. The naming convention makes it possible to mirror one side of the skeleton to the other. The bone orientation allows (future) animations to be mirrored an both sides.
```
